conf/application.conf

```
@@ -36,6 +49,8 @@ play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.http.JsonErrorHandler"
 
 play.modules.enabled += "play.modules.reactivemongo.ReactiveMongoHmrcModule"
 
+lay.modules.enabled += "config.Modules"
```

Should be `play.modules.enabled`?

```suggestion
play.modules.enabled += "config.Modules"
```